operator/src/main/java/oracle/kubernetes/operator/DomainProcessorImpl.java

```diff
@@ -625,10 +626,20 @@ public boolean wasInspectionRun() {
 
     private boolean isShouldContinue() {
       DomainPresenceInfo cachedInfo = getExistingDomainPresenceInfo(getNamespace(), getDomainUid());
+      String existingError = Optional.ofNullable(liveInfo)
```

Comment:
I don't think all introspector errors are reported in the domain status. We need to make sure that all SEVERE JRF ones are.

Reply:
This is one area that I depends on, I believe Anil has fixed this in an earlier PR that makes the SEVER error shows up the domain status.

Reply:
I noticed that the task that tracks Anil's work is still open (https://jira.oraclecorp.com/jira/browse/OWLS-82011).

Reply:
We can wait till 82011 is merged and recheck

Reply:
I am okay with getting this change in and doing more testing once owls-82011 is complete.